### PR TITLE
Coaching profile

### LINKED ISSUE #92: Coach/Judge profiles have weird offsets/little parsing
Only shows name and top information

- https://secure.meetcontrol.com/divemeets/system/profile.php?number=21473
- https://secure.meetcontrol.com/divemeets/system/profile.php?number=16241

### PR DESCRIPTION
**Pull request by spencerdearman**

### CODE REVIEW

File: DiveMeets/DiveMeets/Views/Login/ProfileView.swift [hunk 1]
```diff
@@ -247,13 +259,130 @@ struct ProfileView: View {
...
+            } label: {
+                Text(elements[0])
+                    .fontWeight(.semibold)
+            }
+            
+        }
+    }
+}
+
+struct judgedList: View {
```
Comment (lsherwin10) [inline DiveMeets/DiveMeets/Views/Login/ProfileView.swift:363]: I think the first letter is supposed to be capitalized in structs

File: DiveMeets/DiveMeets/Views/Login/ProfileView.swift [hunk 2]
```diff
@@ -247,13 +259,130 @@ struct ProfileView: View {
...
+                                    current = try t.text()
+                                } else {
+                                    current = try t.text()
+                                }
+                            }
+                        }
+                        if !current.isEmpty {
+                            judgingHistory[current] = eventsList
+                        }
+                    }
```
Comment (lsherwin10) [inline DiveMeets/DiveMeets/Views/Login/ProfileView.swift:304]: Can we move this parsing into a function?

**Review by lsherwin10 — APPROVED** [review]
Just those two comments, looks great

**Comment by spencerdearman** [comment]
For some reason it went to the old name,I am closing this pr and gonna merge the other, it is the exact same save for the poorly chosen name.

**Pull request closed without merging**